tap_github.py

```diff
@@ -254,9 +255,31 @@ def get_all_team_members(team_slug, schemas, repo_path, state, mdata):
                 # transform and write release record
                 with singer.Transformer() as transformer:
                     rec = transformer.transform(r, schemas, metadata=metadata.to_map(mdata))
-                # counter.increment()
+                counter.increment()
                 yield rec
 
+                if schemas.get('team_memberships'):
```

Comment:
I think this code path is never reached because of the yield. This needs to be refactored somehow to actually sync the team memberships.

Reply:
Actually, I think the bigger issue is above this code:

In `get_all_teams`:
```
for team_members_rec in get_all_team_members(team_slug, schemas['team_members'], repo_path, state, mdata):
```

The second arg named `schemas` in `get_all_team_members` does not contain enough information as it gets passed down through the functions. 

It goes from a dictionary of stream-names to schemas to just a single schema. The call to `schemas.get('team_memberships')` always returns None.

Reply:
i see, thanks for pointing that out. i think i can resolve that by doing this:

```
SUB_STREAMS = {
    'pull_requests': ['reviews', 'review_comments'],
    'projects': ['project_cards', 'project_columns'],
    'teams': ['team_members'],
    'team_members': ['team_memberships']
}
```

right?